src/JupyterLibrary/clients/jupyterlab/Selectors.resource

```diff
@@ -38,6 +38,9 @@ ${JLAB XP NB FRAG}                      ${JLAB XP MAIN AREA FRAG}\[contains(@cla
 ${JLAB XP NB TOOLBAR FRAG}              [contains(@class, 'jp-NotebookPanel-toolbar')]
 ${JLAB XP NB TOOLBAR}                   //div${JLAB XP NB TOOLBAR FRAG}
 ${JLAB XP BUSY KERNEL}                  //*[local-name() = 'div' and contains(@class, 'jp-FilledCircleIcon' or (local-name() = 'svg' and contains(@data-icon, 'ui-components:circle-filled')))]
+${JL_TABBAR_CONTENT_XPATH}              //div[contains(@class,"lm-DockPanel-tabBar")]/ul[@class="lm-TabBar-content p-TabBar-content"]
```

Comment:
Please align these with the rest of the naming conventions, e.g. `JLAB XP TAB CONTENT`.

The `lm-` and `p-` classes are precisely equivalent, and `lm-` won't work in jupyterlab 1 (which is still receiving updates). Please continue to use `p-` for now.

Reply:
Done in latest commit